modin/core/execution/dask/common/task_wrapper.py

```diff
@@ -16,11 +16,11 @@
 from distributed.client import default_client
 
 
-class DaskTask:
+class DaskWrapper:
     """The class responsible for execution of remote operations."""
 
     @classmethod
-    def deploy(cls, func, num_returns, kwargs):
+    def deploy(cls, func, num_returns, *args, **kwargs):
```

Comment:
We should probably rename this method to `deploy_func` or `deploy_task`.

Reply:
I'm aligning on Ray for now.

Reply:
Ok, let's do that for all engines at once in a separate PR.